keepsake-app/db.json

```diff
@@ -117,6 +105,72 @@
       ],
       "isChecklist": false,
       "id": "8e94"
+    },
+    {
+      "id": "365b",
+      "userID": 3,
+      "title": "fdg",
+      "content": "broccoli",
+      "category": "note",
+      "labels": [
+        1
```

Comment:
Won't block this PR here - but would be good to have a quick script or something like that resets the state of your `db.json` file after writing it a number of times when testing POST requests